### PR TITLE
ADD: Add catalog translation.

### PR DESCRIPTION
**Pull request by yunyizea**
Added slightly improved Chinese directory translation.

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current.json [hunk 1]
```diff
@@ -4,43 +4,43 @@
...
     "description": "The label for category Setup in sidebar documentationSideBar"
   },
   "sidebar.documentationSideBar.category.Advanced Configuration": {
-    "message": "Advanced Configuration",
+    "message": "KubeEdge 高级配置",
     "description": "The label for category Advanced Configuration in sidebar documentationSideBar"
   },
   "sidebar.documentationSideBar.category.Architecture": {
-    "message": "Architecture",
+    "message": "KubeEdge 结构",
```
Comment (fisherxu) [inline i18n/zh/docusaurus-plugin-content-docs/current.json:15]:
```suggestion
    "message": "KubeEdge 架构",
```

File: i18n/zh/docusaurus-plugin-content-docs/current.json [hunk 2]
```diff
@@ -4,43 +4,43 @@
...
     "description": "The label for category Edge Components in sidebar documentationSideBar"
   },
   "sidebar.documentationSideBar.category.Developer Guide": {
-    "message": "Developer Guide",
+    "message": "KubeEdge 开发者向导",
     "description": "The label for category Developer Guide in sidebar documentationSideBar"
   },
   "sidebar.documentationSideBar.category.Mappers": {
-    "message": "Mappers",
+    "message": "KubeEdge 映射器",
```
Comment (fisherxu) [inline i18n/zh/docusaurus-plugin-content-docs/current.json:31]: No need to translate the `Mapper`, thanks :)

File: i18n/zh/docusaurus-plugin-content-docs/current.json [hunk 3]
```diff
@@ -4,43 +4,43 @@
...
     "description": "The label for category Mappers in sidebar documentationSideBar"
   },
   "sidebar.documentationSideBar.category.Tests": {
-    "message": "Tests",
+    "message": "测试你的 KubeEdge",
```
Comment (fisherxu) [inline i18n/zh/docusaurus-plugin-content-docs/current.json:35]:
```suggestion
    "message": "测试指导",
```

File: i18n/zh/docusaurus-plugin-content-docs/current/roadmap.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 sidebar_position: 3
-title: Roadmap
+title: KubeEdge 的开发路线
```
Comment (fisherxu) [inline i18n/zh/docusaurus-plugin-content-docs/current/roadmap.md:3]:
```suggestion
title: KubeEdge 的开发路标
```

**Comment by fisherxu** [comment]
@yunyizea Thanks for your fixing :)

File: i18n/zh/docusaurus-plugin-content-docs/current.json (hunk 1 above)
Comment (yunyizea) [reply i18n/zh/docusaurus-plugin-content-docs/current.json:15]: I actually translated this at the beginning, and I don't know why I changed back to "结构"    : (

File: i18n/zh/docusaurus-plugin-content-docs/current.json (hunk 2 above)
Comment (yunyizea) [reply i18n/zh/docusaurus-plugin-content-docs/current.json:31]: okay～

File: i18n/zh/docusaurus-plugin-content-docs/current/roadmap.md (hunk 4 above)
Comment (yunyizea) [reply i18n/zh/docusaurus-plugin-content-docs/current/roadmap.md:3]: I translated it into "路线". I think that "开发路线" is more in line with kubeedge's historical map.

**Comment by yunyizea** [comment]
@fisherxu I'm still doing the document translation, thank you for your advice.

**Comment by yunyizea** [comment]
> > I'm still doing the document translation, thank you for your advice.
> 
> You'd better to split the translation work in several PRs, which will be more easy to review and merge :)

Okay, I will~

**Comment by fisherxu** [comment]
Have you already joined the community slack? If not, this is the invitation link, we can have more discussion: https://kubeedge.slack.com/join/shared_invite/enQtNjc0MTg2NTg2MTk0LWJmOTBmOGRkZWNhMTVkNGU1ZjkwNDY4MTY4YTAwNDAyMjRkMjdlMjIzYmMxODY1NGZjYzc4MWM5YmIxZjU1ZDI#/shared-invite/email

**Comment by yunyizea** [comment]
@fisherxu

File: i18n/zh/docusaurus-plugin-content-docs/current.json [hunk 5]
```diff
@@ -4,43 +4,43 @@
...
     "description": "The label for category Cloud Components in sidebar documentationSideBar"
   },
   "sidebar.documentationSideBar.category.Edge Components": {
-    "message": "Edge Components",
+    "message": "KubeEdge 边缘组件",
     "description": "The label for category Edge Components in sidebar documentationSideBar"
   },
   "sidebar.documentationSideBar.category.Developer Guide": {
-    "message": "Developer Guide",
+    "message": "KubeEdge 开发者向导",
```
Comment (Shelley-BaoYue) [inline i18n/zh/docusaurus-plugin-content-docs/current.json:27]: I think "开发者指南" is better. “KubeEdge” can also be omitted here

File: i18n/zh/docusaurus-plugin-content-docs/current/kubeedge.md [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 为什么选择KubeEdge
+title: 为什么选择 KubeEdge
```
Comment (Shelley-BaoYue) [inline i18n/zh/docusaurus-plugin-content-docs/current/kubeedge.md:2]: Why add spaces here?

File: i18n/zh/docusaurus-plugin-content-docs/current.json (hunk 5 above)
Comment (yunyizea) [reply i18n/zh/docusaurus-plugin-content-docs/current.json:27]: ok, I will modify

File: i18n/zh/docusaurus-plugin-content-docs/current/kubeedge.md (hunk 6 above)
Comment (yunyizea) [reply i18n/zh/docusaurus-plugin-content-docs/current/kubeedge.md:2]: I got used to it personally, I'll make the necessary changes right away

**Comment by yunyizea** [comment]
@Shelley-BaoYue Modified

**Comment by kevin-wangzefeng** [comment]
/lgtm
/approve
Thanks

**Pull request merged**